Invalid foreign key exception when bootstrapping SQLite database

### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

```
[...]

12:44:18 ✨ Setting up first admin role...
12:44:18 ✨ Adding first admin user...
12:44:18 ✨ No admin email provided. Defaulting to "<redacted>"
12:44:18 ✨ No admin password provided. Defaulting to "<redacted>"
InvalidForeignKeyException [Error]: Invalid foreign key.
    at Object.extractError (/<redacted>/directus/api/src/exceptions/database/dialects/sqlite.ts:34:10)
    at Object.translateDatabaseError (/<redacted>/directus/api/src/exceptions/database/translate.ts:27:11)
    at /<redacted>/directus/api/src/services/items.ts:139:17 {
  status: 400,
  code: 'INVALID_FOREIGN_KEY',
  extensions: {}
}
npm ERR! Lifecycle script `cli` failed with error:
npm ERR! Error: command failed
npm ERR!   in workspace: directus@9.0.0-rc.89
npm ERR!   at location: /<redacted>/directus/api
```

It does work with Postgres.

Probably related to #7259

### To Reproduce

* Checkout main branch
* Run steps from https://docs.directus.io/contributing/running-locally/, configuring .env file to use SQLite

### What version of Directus are you using?

v9.0.0-rc.89

### What version of Node.js are you using?

v16.6.1

### What database are you using?

SQLite

### What browser are you using?

Firefox

### What operating system are you using?

macOS

### How are you deploying Directus?

Locally

--- a/api/src/services/items.ts
+++ b/api/src/services/items.ts
@@ -134,7 +134,7 @@ export class ItemsService<Item extends AnyItem = AnyItem> implements AbstractSer
 
 			try {
 				const result = await trx.insert(payloadWithoutAliases).into(this.collection).returning(primaryKeyField);
-				primaryKey = result[0];
+				primaryKey = primaryKey ?? result[0];
 			} catch (err) {
 				throw await translateDatabaseError(err);
 			}

